tests/run-make/CURRENT_RUSTC_VERSION/Makefile

```diff
@@ -0,0 +1,6 @@
+include ../tools.mk
+
+all:
+	$(RUSTC) --emit=metadata --crate-type lib stable.rs
+	$(RUSTC) --emit=metadata --extern stable=$(TMPDIR)/libstable.rmeta main.rs 2>&1 >/dev/null \
+		| $(CGREP) -e "stable since $$(cat $(S)/src/version)(-[a-zA-Z]+)?"
```

Comment:
Very nice :heart:

Reply:
I think we support custom metadata in the version string via https://github.com/rust-lang/rust/blob/master/config.example.toml#L557, I'm not quite sure where that goes though. Maybe it doesn't affect this.

Reply:
"it doesn't affect this" is exactly the point of this test - it's ensuring that `CFG_RELEASE` only has the version number, unlike `CFG_VERSION` which includes the custom metadata.